src/common/processUtils.ts

```diff
@@ -0,0 +1,23 @@
+import { Ora } from "ora";
+import * as Sentry from "@sentry/node";
+// @ts-ignore It's necessary for sentry to trace errors. Importing @sentry/tracing patches the global hub for tracing to work.
+import * as Tracing from "@sentry/tracing";
+
+export const isDevelopment = () => process.env.NODE_ENV === "development";
+
+export const setupSentry = () => {
+    if (!isDevelopment()) {
+        Sentry.init({
+            dsn: "https://b4ad6d6db883487ebc914236321bf0e0@sentry.is.canonical.com//50",
+        });
+    }
+} 
+
+export const displayError = (error: Error, spinner: Ora) => {
+    const errorMessage = error.message;
+    errorMessage
+        ? spinner.fail(`${errorMessage}`)
+        : spinner.fail("An error occurred.");
+
+    !isDevelopment() && Sentry.captureException(error);
```

Comment:
Small note, this isn't very readable, I think that something like this is more clear and will help you avoid boolean bugs:
```ts
if(!isDev()) Sentry.capture();
```